src/EndpointManager.sol

```diff
@@ -149,12 +160,27 @@ abstract contract EndpointManager is IEndpointManager, OwnableUpgradeable, Reent
         return amount;
     }
 
+    // @dev Mark a message as executed.
+    // This function will revert if the message has already been executed.
+    function _replayProtect(bytes32 digest) internal {
+        // check if this message has already been executed
+        if (managerMessageAttestations[digest].executed) {
```

Comment:
nit -- reuse `isMessageExecuted(digest)` here?

Reply:
done in https://github.com/wormhole-foundation/example-native-token-transfers/compare/dc94fdb2a22d275407c11dde109f6d8cbcb393bf..84af9b429c9282f8edc910f7d5a2382d58634eed